### ISSUE TITLE
Code on Quick Start file may be mistaken

### ISSUE THREAD

**Issue by menriquez-IDM** [EvoNetHIV/EvoNetHIV#10]
REPRO
=====
- From RStudio,  create a new R project
- Create a new R file.
- Follow all and each of the steps provided on this page:   (this is, install instructions and each of the code provided along the documentation)
- https://github.com/EvoNetHIV/EvoNetHIV/blob/master/documentation/Quick%20start.md 

EXPECTED
=======
After finishing following the steps, the basic sample runs.

ACTUAL
======
After following the steps, the script doesn't work.

**Comment by jtm6** [comment]
Yes, that file is out of date. Look at "default run script" in scripts directory.
